src/app/content/components/ContentLink.tsx

```diff
@@ -59,7 +61,7 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
   const bookUid = stripIdVersion(book.id);
   // Add options only if linking to the same book
   const options = currentBook && currentBook.id === bookUid
-    ? createNavigationOptions(search, scrollTarget)
+    ? createNavigationOptions({...search, ...systemQueryParams}, scrollTarget)
```

Comment:
i don't think `contentlink` should be handling "search" specifically, maybe we can't select persistent params from within here, because maybe we'd be trying to remove them, but we should accept any query params as input to the component, not typed just to search

i would like to have a type somewhere that encapsulates all the possible query params that we support on the content route